src/control/lib/control/printers.go

```diff
@@ -439,6 +439,39 @@ func PrintHostStorageMap(hsm HostStorageMap, out io.Writer, opts ...PrintConfigO
 	return nil
 }
 
+// PrintHostStorageSpaceMap generates a human-readable representation of the supplied
+// HostStorageMap struct and writes utilization info to the supplied io.Writer.
+func PrintHostStorageSpaceMap(hsm HostStorageMap, out io.Writer) error {
```

Comment:
Would be good to add a unit test for this

Reply:
Agreed. All formatters should have unit tests.

Reply:
now tested in cmd/dmg/storage_test.go:TestControl_PrintStorageScanResponse() L279